subsys/net/lib/dns/llmnr_responder.c

```diff
@@ -329,6 +329,7 @@ static int create_ipv4_answer(struct net_context *ctx,
 
```

Comment:
Sanity complains about too long line in commit body, I think you could safely write the offending line like this:
```
.../net/lib/dns/llmnr_responder.c:306:14: note: ‘addr’ was declared here
  const u8_t *addr;
              ^~~~
```

It would make the line a bit shorter.

Reply:
Fixed